ui/desktop/app/routes/scopes/scope/projects/targets.js

```diff
@@ -47,10 +47,17 @@ export default class ScopesScopeProjectsTargetsRoute extends Route {
   @action
   async connect(model) {
     try {
-      await this.ipc.invoke('connect', {
+      await this.ipc.invoke('cli');
```

Comment:
Can the method name be more descriptive?  It's not immediately obvious that this is a check.  Something like `cli-exists'

Reply:
Updated this to 'cliPath' as I'm returning the actual path of cli.